doc/api/deprecations.md

```diff
@@ -534,6 +534,21 @@ deprecated. Please use `ServerResponse.prototype.writeHead()` instead.
 *Note*: The `ServerResponse.prototype.writeHeader()` method was never documented
 as an officially supported API.
 
+<a id="DEP0064"></a>
+### DEP0064: outgoingMessage.\_headers, outgoingMessage.\_headerNames
```

Comment:
If someone tries to land a deprecation with this number before I do, the matter will be settled in an office joust.